Fun.Build/StageContextExtensions.fs

```diff
@@ -205,6 +205,8 @@ module StageContextExtensionsInternal =
 
                     let extraInfo = $"timeout: {timeoutForStage}ms. step timeout: {timeoutForStep}ms."
                     match index with
+                    | StageIndex.WhenStage ->
```

Comment:
I think we can change **WhenStage** to **Condition** for better extension for future.